pkgs/applications/networking/giara/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitLab, lib, pkgs }:
+
+let
+  pname = "giara";
+  version = "0.1.1";
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "GabMus";
+    repo = pname;
+    rev = version;
+    sha256 = "038isdbvax3361kdi2rsgasyic7nmz0l9pd8r531bxfcsqcd1ny8";
+  };
+in
+
+stdenv.mkDerivation {
+  inherit pname version src;
+
+  nativeBuildInputs = with pkgs; [
+    meson pkg-config cmake ninja python3 python3Packages.wrapPython wrapGAppsHook
+  ];
+
+  buildInputs = with pkgs; [
+    gtk3 webkitgtk gtksourceview4 libhandy gobject-introspection glib-networking
+  ];
+
+  pythonPath = with pkgs.python3Packages; [
+    pygobject3 requests dateutil praw pillow mistune flask beautifulsoup4
+  ];
+
+  postInstall = ''
+    wrapPythonPrograms
+  '';
+
+  meta = with lib; {
+    description = "Giara is a reddit app, built with Python, GTK and Handy. Created with mobile Linux in mind.";
```

Comment:
Simplify by removing the project name as per https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes